syft/pkg/license_test.go

```diff
@@ -0,0 +1,96 @@
+package pkg
+
+import (
+	"sort"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/anchore/syft/syft/artifact"
+	"github.com/anchore/syft/syft/source"
+)
+
+func Test_Hash(t *testing.T) {
```

Comment:
this test should be expanded to show that URL and whole locations (not just the FS ID) are not sensitive to the ID calculations